Task_Position_the_Cubes/index.css

```diff
@@ -1,27 +1,71 @@
-
-body {
-  background-color: lightblue;
-}
-p{
```

Comment:
The 'margin' property value should be either auto or specified with a unit such as 'px'